voidcrew/code/datums/ships/_ships.dm

```diff
@@ -0,0 +1,27 @@
+/datum/ship
```

Comment:
If this datum is used for ship parts themselves, shouldn't it be a bit more specific to that?